charts/nginx-ingress/templates/NOTES.txt

```diff
@@ -1 +1,13 @@
-The NGINX Ingress Controller has been installed.
+The NGINX Ingress Controller {{ .Chart.AppVersion }} has been installed.
+
+For release notes for this version please see: https://docs.nginx.com/nginx-ingress-controller/releases/
+
+Installation and upgrade instructions: https://docs.nginx.com/nginx-ingress-controller/installation/installing-nic/installation-with-helm/
+
+{{ if .Release.IsUpgrade -}}
+If you are upgrading from a version of the chart that uses older Custom Resource Definitions (CRD) it is necessary to manually upgrade the CRDs as this is not managed by Helm.
+To update to the latest version of the CRDs:
+  $ kubectl apply -f https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v{{ .Chart.AppVersion }}/deploy/crds.yaml
```

Comment:
is this picking up 3.4.0 or 3.4.2?

Reply:
@vepatel on main it picks up `3.4.0`, but that shouldn't impact released versions, we're just behind on main at the moment